src/App.tsx

```diff
@@ -14,11 +15,39 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  goods: string[],
+  goodsIsVisible: boolean,
```

Comment:
```suggestion
  areGoodsVisible: boolean,
```
множественное число - `are`, не `is`
`are` ставится на первое место

Reply:
you should change name of variable here